File: mi_app/lib/models/person.dart
class Persona {
  int? id;
  bool? status; 
  String? names;
  String? lastName;
  String? documentType;
  String? document;
  // String? documentPhoto;
  String? birthdate;
  String? email;
  bool? gender;

  Persona(
      {
      this.names,
      this.lastName,
      this.documentType,
      this.document,
      this.email
      });

  Persona.fromJson(Map<String, dynamic> json) {
    names = json['names'];
    lastName = json['lastName'];
    documentType = json['documentType'];
    document = json['document'];
    email = json['email'];
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = new Map<String, dynamic>();
    data['names'] = this.names;
    data['lastName'] = this.lastName;
    data['documentType'] = this.documentType;
    data['document'] = this.document;
    data['email'] = this.email;
    return data;
  }
}

File: CodeByte/src/main/java/com/ProyectoGT/CodeByte/Entity/Security/Person.java
package com.ProyectoGT.CodeByte.Entity.Security;

import java.time.LocalDateTime;

import com.ProyectoGT.CodeByte.Entity.Base;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.Table;

@Entity
@Table(name = "person")
public class Person extends Base{

	@Column(name = "Names",nullable = true, length = 50)
	private String names;
	
	@Column(name = "last_name",nullable = true, length = 50)
	private String lastName;
	
	@Column(name = "document_type",nullable = true, length = 50)
	private String documentType;
	
	@Column(name = "document",nullable = true, length = 50)
	private String document;
	
	@Column(name = "document_photo",nullable = true, columnDefinition = "TEXT")
	private String documentPhoto;
	
	@Column(name = "birthdate",nullable = true, length = 50)
	private LocalDateTime birthdate;
	
	@Column(name = "email",nullable = true, length = 100)
	private String email;
	
	@Column(name = "gender",nullable = true, length = 50)
	private boolean gender;

	public String getNames() {
		return names;
	}

	public void setNames(String names) {
		this.names = names;
	}

	public String getLastName() {
		return lastName;
	}

	public void setLastName(String lastName) {
		this.lastName = lastName;
	}

	public String getDocumentType() {
		return documentType;
	}

	public void setDocumentType(String documentType) {
		this.documentType = documentType;
	}

	public String getDocument() {
		return document;
	}

	public void setDocument(String document) {
		this.document = document;
	}

	public String getDocumentPhoto() {
		return documentPhoto;
	}

	public void setDocumentPhoto(String documentPhoto) {
		this.documentPhoto = documentPhoto;
	}

	public LocalDateTime getBirthdate() {
		return birthdate;
	}

	public void setBirthdate(LocalDateTime birthdate) {
		this.birthdate = birthdate;
	}

	public String getEmail() {
		return email;
	}

	public void setEmail(String email) {
		this.email = email;
	}

	public boolean isGender() {
		return gender;
	}

	public void setGender(boolean gender) {
		this.gender = gender;
	}
	
	
}
